doc/collision_detection.md

```diff
@@ -25,6 +25,10 @@ the latter is very useful for accurate gesture detection. The collision detectio
 what should happen when two hitboxes collide, so it is up to the user to implement what will happen
 when for example two position components have intersecting hitboxes.
 
+Do note that the built-in collision detection system does not take collisions between two hitboxes
+that overshoot each other into account, this could happen when they either move too fast or `update
```

Comment:
there is an extra backtick here

Reply:
also we can mention this is normally called tunneling, there are a lot of great articles around about it and how to solve it

Reply:
Fixed